app/questionnaire/questionnaire_schema.py

```diff
@@ -1385,27 +1385,48 @@ def get_item_anchor(self, section_id: str, list_name: str) -> str | None:
                 if item["for_list"] == list_name and item.get("item_anchor_answer_id"):
                     return f"#{str(item['item_anchor_answer_id'])}"
 
+    def _update_placeholder_dependencies(
+        self, placeholder_answer_ids: set, block: ImmutableDict
+    ) -> None:
+        if placeholder_dependencies := self._get_section_ids_for_answer_ids(
```

Comment:
Existing, but it would be clearer if this was named `dependent_sections`

Reply:
Updated 👍